Libs/MRML/Logic/vtkMRMLColorLogic.h

```diff
@@ -189,7 +390,17 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLColorLogic : public vtkMRMLAbstractLogic
   /// vtkMRMLApplication::GetColorFilePaths
   char *UserColorFilePaths;
 
+  // mappings used for terminology color look ups
+  typedef std::map<int,ColorLabelCategorization> ColorCategorizationMapType;
+  std::map<std::string, ColorCategorizationMapType> colorCategorizationMaps;
+
   static std::string TempColorNodeID;
+
+ private:
+
+  std::string RemoveLeadAndTrailSpaces(std::string);
+  bool ParseTerm(std::string, StandardTerm&);
```

Comment:
Consider moving this as protected. That way in you test, you can derive `vtkMRMLColorLogic` into `vtkMRMLTestColorLogic` and test the method. For example, see https://github.com/jcfr/Slicer/commit/f2aa068e24277227931ca25049a970ce25429cba#diff-7aa2fc5db0ae46d14ee8f13646159833